tests/provision/local/test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
+        rlRun "pushd data"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Test script installation"
+        # Run tmt with local provision
+        rlRun "tmp_root=\$(mktemp -d)"
+        rlRun -s "tmt --feeling-safe run --workdir-root $tmp_root"
```

Comment:
```suggestion
        rlRun -s "tmt --feeling-safe run --id $run -vvv"
```

The workdir was needed before because we installed scripts directly there. Now that we put them under run workdir we don't need a dedicated workdir. Let's:

        rlRun "run=\$(mktemp -d)" 0 "Create run directory"

... in the setup phase instead and use it directly.